pyproject.toml

```diff
@@ -1,6 +1,98 @@
 [build-system]
-requires = ["setuptools", "setuptools_scm", "wheel"]
-build-backend = 'setuptools.build_meta'
+requires = [
+  "setuptools>=62.1",
+  "setuptools_scm[toml]>=6.2",
+  "wheel",
 
-[tool.black]
-line-length = 101
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "parfive"
+description = "A HTTP and FTP parallel file downloader."
+readme = "README.rst"
+requires-python = ">=3.7"
+license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+
+authors = [
+  { name = "Stuart Mumford", email = "stuart@cadair.com" },
+]
+
+dependencies = [
+
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-doctestplus",
+]
+docs = [
+    "sphinx",
+    "sphinx-automodapi",
+    "tomli; python_version <\"3.11\"",
+]
+
+
+
+
+[project.urls]
+repository = "https://parfive.readthedocs.io/"
```

Comment:
This is wrong, looks like a template bug

Reply:
thats the url to the docs not the repo

Reply:
I'm not sure what's happened here, all the template stuff looks to be in good order, did I just mess up the input?

Reply:
It should be `Homepage =`